libmamba/include/mamba/api/configuration.hpp

```diff
@@ -317,6 +317,28 @@ namespace mamba
         };
 
 
+        template <>
+        struct cli_config<std::uint16_t>
```

Comment:
in order to keep binary size smaller, why not use std::size_t or another type that's already being used?

Reply:
yes we can do that